pyproject.toml

```diff
@@ -50,6 +50,7 @@ dependencies = [
     'sqlalchemy>=2.0',  # GeoAlchemy2 requires >=1.4.
     'toolz',
     'xarray>=0.9',  # >0.9 fixes most problems with `crs` attributes being lost
+    'odc-stac'      # brings in odc-geo and odc-loader
```

Comment:
Is any version of odc-stac fine, or does it need to be a recent version?

Reply:
Really needs to be <=0.4.0 as that's when it started using odc-loader externally.